docs/docs/segment-owm.md

```diff
@@ -29,7 +29,8 @@ The free tier for *Current weather and forecasts collection* is sufficient.
     "location": "AMSTERDAM,NL",
     "units": "metric",
     "enable_hyperlink" : false,
-    "http_timeout": 20
+    "http_timeout": 20,
+    "cachefile": ""
```

Comment:
Why don't we make use of for example writing an environment variable to store this context?

Reply:
Using one or two environment variables instead of a file is a good idea! I'll try to get back to it this weekend. Thanks for the suggestions!

Reply:
the only upside of the temp file is when you use multiple console at the same time.

Reply:
@lnu in this use-case I feel the cache can be the same cross shell for this one.

Reply:
So I dug into this a little bit last night and if you set an environment variable from within the application, the value is not persisted outside the run of the application.

At least not when using os.Setenv() - I don't think this will work since the OMP process isn't long lived, it's just run long enough to get the prompt text.

Any other thoughts on how to do this without requiring the user to specify a file location?

Reply:
You're right, you can only set a variable for a process and its children... so it's not possible to store it in an env variable.

Reply:
I like tempdir. We would need to create a cache mechanism rather than implement it solely here though. Ideally key-value based.

Reply:
good idead for the global cache, in the meantime I did some changes based on the branch here(https://github.com/lnu/oh-my-posh/tree/feat/owm_cache)

Reply:
#982 has a an implementation idea for a persisted cache